src/main.ts

```diff
@@ -1,10 +1,11 @@
 import { NestFactory } from '@nestjs/core';
 import { config, DotenvConfigOutput } from 'dotenv';
 import * as cookieParser from 'cookie-parser';
+import * as path from 'path';
 
 import { AppModule } from './app.module';
 
-const results: DotenvConfigOutput = config();
+const results: DotenvConfigOutput = config({path: path.resolve(__dirname, '..', '.env')});
```

Comment:
Or, in plain English: look in whatever directory is one higher than `main.js` for the `.env` file.